yamllint/config.py

```diff
@@ -111,6 +113,9 @@ def parse(self, raw_content):
             self.yaml_files = pathspec.PathSpec.from_lines('gitwildmatch',
                                                            conf['yaml-files'])
 
+        if 'locale' in conf:
+            self.locale = conf['locale']
```

Comment:
Can you add a small check to `raise YamlLintConfigError(...)` if `conf['locale']` is not a string?